packages/transformers/babel/src/babel7.js

```diff
@@ -45,6 +45,7 @@ export default async function babel7(
       name: 'parcel',
       version: transformerVersion,
       targets: JSON.stringify(babelOptions.targets),
+      env: asset.env.outputFormat,
```

Comment:
We should call this `outputFormat` since we aren't passing the entire `env` object